pipeline/cactus_workflow.py

```diff
@@ -440,50 +510,50 @@ def inverseJukesCantor(d):
     assert d >= 0.0
     return 0.75 * (1 - math.exp(-d * 4.0/3.0))
     
-class CactusCafPhase(CactusPhasesTarget):      
-    def run(self):
+class CactusCafPhase(CactusPhasesJob):      
+    def run(self, fileStore):
         if (not self.cactusWorkflowArguments.configWrapper.getDoTrimStrategy()) or (self.cactusWorkflowArguments.outgroupEventNames == None):
             setupFilteringByIdentity(self.cactusWorkflowArguments)
         #Setup any constraints
         if self.getPhaseIndex() == 0 and self.cactusWorkflowArguments.constraintsFile != None: #Setup the constraints arg
-            newConstraintsFile = os.path.join(self.getGlobalTempDir(), "constraints.cig")
+            newConstraintsFile = fileStore.getLocalTempFile()
```

Comment:
You can just remove the constraints argument if you want. I don't think it's even usable at all at this point, since the sequence names would have to be the cactus internal names, which are just arbitrary integers.